CoastalResilienceNetwork/GeositeFramework
# Update styleguide colors and incorporate new style guide created by @cschneebeck

This PR does a few things related to the `styleguide` app, and the framework itself:

- The framework DOM and stylesheet used in the `styleguide` were out-of-date with the app. They are updated to the latest here.
- @cschneebeck's style guide was added to the `styleguide` app and is accessible at `http://localhost:3000/snippet_index.html` when running the `styleguide` dev server.
- Components added in the new style guide were added to the existing style guide.
- Some of the base colors used in `styleguide` were updated per #875.
- The framework was updated with the latest version of the `styleguide` app stylesheet and the new CSS and JS dependencies needed by the new components added in @cschneebeck's style guide.

*Existing style guide, new components, link to new style guide*
![image](https://cloud.githubusercontent.com/assets/1042475/23186627/d1283e14-f855-11e6-8bab-3e258aefc17b.png)

*New style guide, with link to existing style guide*
![image](https://cloud.githubusercontent.com/assets/1042475/23186685/ee7d4496-f855-11e6-9d30-d2d1e85c6938.png)


**Testing**
- Run the `styleguide` app and verify that 1) the existing styleguide has the new components, 2) the styleguides are linked, and 3) the new styleguide works.
- Verify the color changes in 21c8345 satisfy #875 and are effective.
- Verify that the framework has been updated with the latest stylesheets and dependencies.

Connects to #883 
Connects to #875
